Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -340,41 +335,43 @@ internal final class AddNewCardViewModelTests: TestCase {
 
   func testUnsupportedCardMessage_showsWithInvalidCardBrand_AndExistingCardNumber() {
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: .pledge, project: Project.template)
 
     self.creditCardValidationErrorContainerHidden
       .assertValues([true], "Unsupported card message is hidden on viewDidLoad")
 
-    self.vm.inputs.cardBrand(isValid: false)
-    self.vm.inputs.creditCardChanged(cardDetails: ("123", nil, nil, nil))
+    self.vm.inputs.creditCardChanged(cardDetails: ("123", nil, nil, nil, .generic))
 
     self.creditCardValidationErrorContainerHidden
       .assertValues([true, false], "Unsupported card message shows")
   }
 
   func testUnsupportedCardMessage_hidesWithValidCardBrand_AndExistingCardNumber() {
+    let project = Project.template
+      |> Project.lens.location .~ .australia
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: .pledge, project: project)
 
     self.creditCardValidationErrorContainerHidden
       .assertValues([true], "Unsupported card message is hidden on viewDidLoad")
 
-    self.vm.inputs.cardBrand(isValid: true)
-    self.vm.inputs.creditCardChanged(cardDetails: ("123", nil, nil, nil))
+    self.vm.inputs.creditCardChanged(cardDetails: ("424", nil, nil, nil, .visa))
 
     self.creditCardValidationErrorContainerHidden
       .assertValues([true, true], "Unsupported card message hides with a valid card brand")
   }
 
   func testUnsupportedCardMessage_hidesWithEmptyOrInvalidCardNumber() {
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: .settings, project: nil)
 
     self.creditCardValidationErrorContainerHidden
       .assertValues([true], "Unsupported card message is hidden on viewDidLoad")
 
-    self.vm.inputs.cardBrand(isValid: false)
-    self.vm.inputs.creditCardChanged(cardDetails: ("", nil, nil, nil))
+    self.vm.inputs.creditCardChanged(cardDetails: ("", nil, nil, nil, nil))
```

Comment:
A better test here would be to call this input with `cardDetails: ("1", nil, nil, nil, .generic)` since we're trying to verify that even is a card brand is invalid (as `.generic` is), the error message should hide if the card number character count is < 2.